docs/Best practices/Type licenties.md

```diff
@@ -3,27 +3,101 @@ De documentatie is nog volop in ontwikkeling
+onder de GPLv3 licentie valt. Component A en component B kunnen nog steeds wel onafhankelijk hergebruikt worden onder 
+hun oorspronkelijke licentie. Strong copyleft licenties zijn dus niet verenigbaar met gesloten software. Het bekendste 
+voorbeeld is: GPLv3.
 
-### Reciprocal / weak copyleft (zwak wederkerig): 
-De gebruiker van de Open Source software mag de software aanpassen naar eigen inzicht aanpassen, maar moet wel bij verspreiding van de software de originele code mee beschikbaar stellen inclusief de aanpassen daarop. Ook hier ben je vaak verplicht om een verwijzing naar de oorspronkelijk code te maken. Bekende voorbeelden zijn: Mozilla Public License.
+Een variant van de GPLv3 is de LGPLv3. De LGPL vermindert het virale effect door een totale software oplossing niet 
+te infecteren met de licentie van het hergebruikte component wanneer het als library (soft-linked) wordt gekoppeld.
 
-### Restricted / strong copyleft (sterk wederkerig): 
-Dit is het soort licentie waaronder je het minst geheim mag houden en meest openbaar moet delen. De gebruiker van de open source software mag de software aanpassen naar eigen inzicht aanpassen, maar moet bij verspreiding van de software de originele code mee beschikbaar stellen inclusief de aanpassen daarop. Ook je eigen code die integreert met deze broncode moet openbaar beschikbaar gesteld worden. Bekende voorbeelden zijn: GNU General Public License en GNU Library General Public License.
+### Noten
 
-Een Open Source licentie uit de smaak ‘Restricted / strong copyleft’ vergt meer kennis van deze betreffende Open Source licentie om deze juist te kunnen toe te passen. Als je gebruikt maakt van Open Source software met een Open Source licentie uit deze smaak dien je met meer voorwaarden rekening te houden. Wanneer je onjuist rekening houd met deze voorwaarden loop je het risico dat je aangesproken / aangeklaagd wordt door de oorspronkelijke maker voor onjuist gebruik.
+Een OSS licentie uit de smaak ‘Restricted / strong copyleft’ vergt meer kennis van deze betreffende OSS licentie om 
+deze juist te kunnen toe te passen. Als je gebruikt maakt van OSS met een OSS licentie uit deze smaak dien je met meer 
+voorwaarden rekening te houden. Wanneer je onjuist rekening houd met deze voorwaarden loop je het risico dat je 
+aangesproken / aangeklaagd voor auteursrechtschending door de oorspronkelijke maker.
+
+Met een OSS licentie geef je de controle uit handen op welke manier je broncode gebruikt gaat worden in nieuwe 
+toepassingen. Je broncode kan dus gebruikt worden voor de ontwikkeling van levensreddende medicijnen, maar net zo goed 
+in een kernwapen waarmee de mensheid wordt vernietigd.
+
+Verder is het goed om te realiseren dat een OSS licentie geen openbaarheid afdwingt. Wat een OSS licentie afdwingt is 
+het recht van vrije distributie. Dat recht wordt pas actief wanneer er sprake is van daadwerkelijke distrubitie oftewel 

```

Comment:
Wellicht goed hier te noemen dat het recht van vrije distributie is een van de zaken die OS licentie afdwingt. Vrij gebruik en vrij aanpassen zijn ook rechten die een OS licentie afdwingt.

Reply:
Ik zie nu ook een spelfout in `distrubitie` 😄 

> Wellicht goed hier te noemen dat het recht van vrije distributie is een van de zaken die OS licentie afdwingt. 

Bedoel je dat nog anders dan dat ik het hier beschreven heb?

> Vrij gebruik en vrij aanpassen zijn ook rechten die een OS licentie afdwingt.

Prima